src/internal/m365/collection/groups/backup.go

```diff
@@ -147,7 +148,7 @@ func populateCollections(
 
 		// ictx = clues.Add(ictx, "previous_path", prevPath)
 
-		items, _, err := bh.getChannelMessagesDelta(ctx, cID, "")
+		items, _, err := bh.getChannelMessageIDsDelta(ctx, cID, "")
```

Comment:
Ideally we should get IDs and modtimes so that we can send enough data to trigger kopia assisted incremental backups without having to fetch the full item. The modtime returned for that should be the same modtime we eventually return for backup details. To simplify the logic a bit, it may be best to fetch all fields that are required in backup details here (maybe with the exception of size)

I'm not sure if I'm looking at the correct code, but it does seem like this call is getting all fields during the delta query under the hood

Reply:
> Ideally we should get IDs and modtimes so that we can send enough data to trigger kopia assisted incremental backups without having to fetch the full item

Sounds good.  I'll take care of this in a follow-up PR.

> I'm not sure if I'm looking at the correct code, but it does seem like this call is getting all fields during the delta query under the hood

You're looking in the right place.  Property selection isn't supported for items at this time.  We could optimize around passing the entire item, if wanted.  I opted not to at this time.